INSTALL

```diff
@@ -211,64 +193,74 @@ IP addresses:
 >>>>> (end of commands)
 
 
-6. Install Eucalyptus (on all systems)
-=====================
+6. Install and Configure Eucalyptus (on all systems)
+===================================
 
 You can now install the Eucalyptus RPMs onto each system.
+The Eucalyptus documentation can be found here:
+http://docs.hpcloud.com/eucalyptus
 
-Follow the instructions from the Installation Guide for all of the steps to
-plan and configure your system's dependencies, before the steps describing
-how to install Eucalyptus or nightly release packages. The Eucalyptus
-documentation can be found here:
+Follow the instructions in the Installation Guide for the following
+sections:
 
-http://docs.hpcloud.com/eucalyptus
+- Plan Your Installation
```

Comment:
Maybe you could have the markdown equivalent of a link here, or just have it in parenthesis.  Do you think most people use the text version of this file or read it on the web?